src/plugins/source-plugin.js

```diff
@@ -582,11 +586,49 @@ export default (options) =>
                   parser.startIndex,
                   parser.endIndex,
                   html
-                ),
+                )
               });
 
               return;
             }
+            if (attribute.toLowerCase() === 'style') {
+              const urlSources = [];
+
+              source.value.split('url(').forEach((urlSource, index) => {
+                /* Ignoring first index because the first is just the css propertie for example:
+                [0:'background-image: ', 1:'
+                "image.png");\ntext-align: center;text-decoration: overline; color:red;background-image:',
+                '"image.png")']*/
+
+                if (index !== 0) {
+                  // removing the rest of the url value ');....' and removing single and douple quotations
+                  urlSource = urlSource
+                    .substring(0, urlSource.indexOf(')'))
+                    .replace(/["']/g, '');
+                  urlSources.push(urlSource);
+                }
+              });
+              let currIndex = 0;
+
+              urlSources.forEach(urlSource => {
+                if (!urlFilter(attribute, urlSource, resourcePath)) {
+                  return;
+                }
+
+                // currIndex to indicate to the current urlSource
+                const sourceStartIndex = source.value.indexOf(
+                  urlSource,
+                  currIndex
+                );
+
+                const startIndex = valueStartIndex + sourceStartIndex;
+                // incrementing currIndex so we can go to the next urlSource
+                currIndex = sourceStartIndex + 1;
+
+                sources.push({ startIndex, value: urlSource, unquoted });
+              });
+              return;
+            }
```

Comment:
Very bad idea, we should use `css-loader` for that:
1. Introduce new attribute type: `type: src | srcset | style`
2. Run `css-loader` for `style`
3. Stringify result

Ideally we should have two tests: 
- only css-loader
- css-loader +postcss-loader, it is allow to autoprefixing in `style` attribute

Reply:
@evilebottnawi I couldn't find a way to run css-loader on a string 
all I can think about doing is to export style content to temporary file tmp.css and then find a way to run webpack with css-loader on that file, this will make `html-loader` depend on `css-loader` and `webpack`.
Is there a way to run css-loader only on a string ?

Reply:
This is where we should think. In theory we should create virtual module. Maybe we right now we can use `css-loader.call(content)`, maybe we need more options/API for that.

Reply:
I need time to think about it.